### PR TITLE
added SUPPORT.md and SECURITY.md (as requested in issue #1)

### PR DESCRIPTION
**Pull request by phschimm**
adjusted CODE_OF_CONDUCT.md to act as a template

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Contributor Covenant Code of Conduct
 
+> __PLEASE NOTE:__  
+> This document is the Official Covenant Code of Conduct that must be present in the root folder of _every_ devonfw project as `CODE_OF_CONDUCT.md`. Please, include this file (minus this note) in your repository and add the product owners email address in the place marked below.
+
```
Comment (hohwille) [inline CODE_OF_CONDUCT.md]: I do not get it. The idea of this .github repo is exactly that we do *not* have to copy the Code of Conduct to every repository as it is inherited automatically to all repos. Therefore this section does not make any sense to me.

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Support
+
+We strive to support all community members and users of devonfw as best as we can, weather that be in their efforts to contribute to the framework directly, or to develop applications based on the framework.  
+Please don't hesitate to contact us via [Microsoft Teams](https://teams.microsoft.com/l/team/19%3af92c481ec30345a28a5434bc530a882a%40thread.skype/conversations?groupId=503df57a-d454-4eec-b3bc-d6d87c7c24f8&tenantId=76a2ae5a-9f00-4f6b-95ed-5d33d77c4d61) / [Yammer](https://www.yammer.com/capgemini.com/#/threads/inGroup?type=in_group&feedId=5030942) or via E-Mail at <icsddevonfwsupport.apps2@capgemini.com>.
```
Comment (hohwille) [inline SUPPORT.md]: I am not quite sure if we can and should to place our company internal channels in this public place. Providing the CG support Email is fine to me, but the other things need to be discussed with @sjimenez77 and others...

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (phschimm) [reply CODE_OF_CONDUCT.md]: Ok, I'll remove this notice and just add our distribution list e-mail at the marked spot. Would that be o.k.?

File: SUPPORT.md (hunk 2 above)
Comment (phschimm) [reply SUPPORT.md]: I wasn't sure weather I sould include them either, but it seemed to be so little content to just have the CG support e-mail in here (also kind of redundant if we include it in `CODE_OF_CONDUCT.md`).

That's the same problem I'd have with a potential `SECURITY.md`; is it really worth bloating our repos with these default files, if they just contain a short "marketing pitch" and not much specific info?

Comment (sjimenez77) [reply SUPPORT.md]: It is not possible for outside Capgemini collaborators to contact us via MS Teams , so from my point of view we should only add email addresses or suggest the corresponding GitHub issue space for each asset.

**Pull request merged**